packages/core/lib/modules/wallet.js

```diff
@@ -41,14 +43,22 @@ export class Wallet {
   networkManager: NetworkManager;
   context: string[];
   status: WalletStatus;
+  apiConnected: boolean;
   eventManager: EventManager;
-
-  constructor() {
-    this.context = ['https://w3id.org/wallet/v1'];
+  walletId: string;
+  accounts: Accounts;
+
+  constructor({
+    walletId = 'dock-wallet',
```

Comment:
does always hard-coding this to the same thing open the possibility of wallet collisions later on during imports or something if an existing (but different) wallet with the same id already exists in that environment?

Reply:
dock-wallet is the default value, but the user can override it. Using an existing id will not override the existing data, it will just load the existing wallet instead.